usaspending_api/awards/models_matviews.py

```diff
@@ -334,3 +334,318 @@ class SummaryTransactionMonthView(models.Model):
     class Meta:
         managed = False
         db_table = 'summary_transaction_month_view'
+
+
+class AwardMatview(models.Model):
+    generated_unique_award_id = models.TextField(blank=True, null=True)
```

Comment:
This col should never be null, but does it make sense to make every single col completely accurate? It might be better just to remove the kwargs from these classes since Django won't manage/enforce them. And future devs won't be confused by mismatched definitions

Reply:
I like the suggestion of removing the kwargs on unmanaged models like this. Let's move forward with that. Will make that change.